vms/platformvm/txs/tx.go

```diff
@@ -31,7 +31,7 @@ type Tx struct {
 	// The credentials of this transaction
 	Creds []verify.Verifiable `serialize:"true" json:"credentials"`
 
-	id    ids.ID
+	MyID  ids.ID `json:"id"`
```

Comment:
Same question on txIDs as for the X-chain here

Reply:
same answer